### PR TITLE
[Feature][Sentry Sink V2] Add Sentry Sink Option Rules

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.sentry.sink;
+
+import org.apache.seatunnel.api.configuration.util.OptionRule;
+import org.apache.seatunnel.api.table.factory.TableSinkFactory;
+
+public class SentrySinkFactory implements TableSinkFactory {
```
Comment (ashulin) [inline seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java:27]:
```suggestion
@AutoService(Factory.class)
public class SentrySinkFactory implements TableSinkFactory {
```

Comment (CalvinKirs) [reply seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java:27]: Hi, @EricJoy2048 It looks like this issue has not been resolved

Comment (EricJoy2048) [reply seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java:27]: > Hi, @EricJoy2048 It looks like this issue has not been resolved

After discuss, we need use `@AutoService(TableSinkFactory.class)` replace `@AutoService(Factory.class)`

Comment (EricJoy2048) [reply seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java:27]: @CalvinKirs  replace to `@AutoService(Factory.class)` done.

**Review by Hisoka-X — APPROVED** [review]

**Review by ic4y — COMMENTED** [review]
+1

**Review by ic4y — APPROVED** [review]

**Pull request merged**